storage/aptosdb/src/rocksdb_property_reporter.rs

```diff
@@ -139,7 +139,7 @@ fn update_rocksdb_properties(
             set_property(cf, ledger_db.transaction_accumulator_db_raw())?;
         }
 
-        for cf in state_kv_db_column_families() {
+        for cf in state_kv_db_new_key_column_families() {
```

Comment:
We just break the dashboard for the old format?

Reply:
updated. Probably need to update dashboard to show different CFs when in migration